src/components/ChangeData/ChangeProfile.module.scss

```diff
@@ -39,8 +43,32 @@
   margin-bottom: 1rem;
   border: none;
   border-bottom: 1px solid $translucent-silk;
+  color: $translucent-silk;
   font-size: 1rem;
   font-weight: 700;
+
+  &:focus {
+    outline: 1px solid $translucent-silk;
+  }
+
+  @media (min-width: $screen-md-min) {
+    font-size: 1.2rem;
+  }
+}
+
+.textField::placeholder {
+  color: #c5bebe7e;
```

Comment:
Move to color file. No one-off colors are tolerated :D

Reply:
DONE ✅